download/download.html.haml

```diff
@@ -117,6 +117,22 @@ title: Download jBPM
 
       * http://registry.hub.docker.com/u/jboss/jbpm-workbench/[jBPM Workbench]
       * http://registry.hub.docker.com/u/jboss/jbpm-workbench-showcase/[jBPM Workbench Showcase]
-      * http://registry.hub.docker.com/u/jboss/jbpm-server-full/[jBPM Server Full (zip)]
+      * http://registry.hub.docker.com/u/jboss/jbpm-server-full/[jBPM Server Full]
```

Comment:
I'd actually put the server full as first one and the other two below (even in separate section if at all) - why we have two workbenches and no KIE Server?

Reply:
Sure, changed the order and added link to kie server image.